src/components/selection-panel/selection-panel.ts

```diff
@@ -151,22 +147,16 @@ export class SbbSelectionPanelElement extends LitElement {
     );
     this.addEventListener('checkboxLoaded', () => this._updateSelectionPanel(), { signal });
     this.addEventListener('radioButtonLoaded', () => this._updateSelectionPanel(), { signal });
-    this._handlerRepository.connect();
   }
 
   protected override firstUpdated(): void {
     this._didLoad = true;
   }
 
-  public override disconnectedCallback(): void {
-    super.disconnectedCallback();
-    this._handlerRepository.disconnect();
-  }
-
   private _updateSelectionPanel(): void {
     this._checked = this._input?.checked;
     this._state =
-      this._checked || this.forceOpen || !this._namedSlots['content'] ? 'opened' : 'closed';
+      this.forceOpen || (this._namedSlots['content'] && this._checked) ? 'opened' : 'closed';
```

Comment:
Should not `has()` be used here?
```suggestion
      this.forceOpen || (this._namedSlots.slots.has('content') && this._checked) ? 'opened' : 'closed';
```

Reply:
this is bad :| 
I'll blame the holidays for it